blocks/tabs/tabs.js

```diff
@@ -42,6 +43,10 @@ export default async function decorate(block) {
     tabpanel.setAttribute('aria-hidden', !!i);
     tabpanel.setAttribute('aria-labelledby', `tab-${id}`);
     tabpanel.setAttribute('role', 'tabpanel');
+    if (block.classList.contains('vertical') && block.classList.contains('video')) {
+      const wistiaLink = tabpanel.querySelector('a');
+      tabpanel.replaceChildren(embedWistia(wistiaLink));
```

Comment:
Didn't dig deep enough yet, but I imagine this could be optimized to lazy load non-visible tabs to gain performance. The TBT looks awful on this

Reply:
Hey @kronnox nice; suggestion; I just applied lazy load on all of the video embeds!